reactor-netty-http/src/main/java/reactor/netty/http/server/DefaultHttpForwardedHeaderHandler.java

```diff
@@ -138,7 +138,11 @@ private static String parseForwardedPrefix(String prefixHeader) {
 			}
 			prefix.append((endIndex != rawPrefix.length() ? rawPrefix.substring(0, endIndex) : rawPrefix));
 		}
-		return prefix.toString();
+		String parsedPrefix = prefix.toString();
+		if (!parsedPrefix.isEmpty() && DEFAULT_FORWARDED_HEADER_VALIDATION && !parsedPrefix.startsWith("/")) {
```

Comment:
instead of `startsWith("/")` what about `tempUri.charAt(0) != '/'`

Reply:
Yep, that's better. Applied the change.